packages/round-manager/src/features/round/RoundDetailForm.tsx

```diff
@@ -56,7 +56,14 @@ export const RoundValidationSchema = yup.object().shape({
         })
         .when("type", {
           is: (val: string) => val && val != "Email",
-          then: yup.string().url().required("You must provide a valid URL."),
+          then: yup
+            .string()
+            /*Matches www.example.com, example.com, http and https prefixes, but not www.invalid */
+            .matches(
+              /^(http:\/\/|https:\/\/|ipfs:\/\/)?\S+\.\S+$|^(ipfs:\/\/)\S+$/,
```

Comment:
This does not allow for URLs not ending after the TLD? `https://blabla.com/ciao` would not pass if I'm reading this correctly. Should we allow paths in the URL?

Reply:
According to regex101, paths in the url are fine.